rpcserver.go

```diff
@@ -3013,14 +3013,33 @@ func SignRawTransaction(w *wallet.Wallet, chainSvr *chain.Client, icmd interface
 			err = vm.Execute()
 		}
 		if err != nil {
-			signErrors = append(signErrors,
-				btcjson.SignRawTransactionError{
-					TxID:      txIn.PreviousOutPoint.Hash.String(),
-					Vout:      txIn.PreviousOutPoint.Index,
-					ScriptSig: hex.EncodeToString(txIn.SignatureScript),
-					Sequence:  txIn.Sequence,
-					Error:     err.Error(),
-				})
+			multisigNotEnoughSigs := false
+			class, addr, _, _ := txscript.ExtractPkScriptAddrs(
+				inputs[txIn.PreviousOutPoint],
+				activeNet.Params)
+
+			if err == txscript.ErrStackUnderflow &&
+				class == txscript.ScriptHashTy {
+				redeemScript, _ := getScript(addr[0])
```

Comment:
Oh well nevermind, you checked that it's a `ScriptHashTy` already, so the address will exist.